src/ray/common/task/task_spec.cc

```diff
@@ -46,7 +46,7 @@ SchedulingClass TaskSpecification::GetSchedulingClass(
   if (it == sched_cls_to_id_.end()) {
     sched_cls_id = ++next_sched_id_;
     // TODO(ekl) we might want to try cleaning up task types in these cases
-    if (sched_cls_id > 100) {
+    if (sched_cls_id > 100 && sched_cls_id <= 1000) {
```

Comment:
actually this will be really spammy maybe we should print once in 1 minute or just use warning message